From 2c092c99dd959ed751101877102deebbdbab481a Mon Sep 17 00:00:00 2001
From: programminghoch10 <16062290+programminghoch10@users.noreply.github.com>
Date: Mon, 27 Jan 2025 11:29:44 +0100
Subject: [PATCH] remove CheckEnvironmentPatch

since we build on device this patch is not required
---
 .../revanced/patches/youtube/misc/settings/SettingsPatch.kt    | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/patches/src/main/kotlin/app/revanced/patches/youtube/misc/settings/SettingsPatch.kt b/patches/src/main/kotlin/app/revanced/patches/youtube/misc/settings/SettingsPatch.kt
index fce41f327..4bf9552ce 100644
--- a/patches/src/main/kotlin/app/revanced/patches/youtube/misc/settings/SettingsPatch.kt
+++ b/patches/src/main/kotlin/app/revanced/patches/youtube/misc/settings/SettingsPatch.kt
@@ -181,9 +181,6 @@ val settingsPatch = bytecodePatch(
         addResourcesPatch,
         versionCheckPatch,
         fixPlaybackSpeedWhilePlayingPatch,
-        // Currently there is no easy way to make a mandatory patch,
-        // so for now this is a dependent of this patch.
-        checkEnvironmentPatch,
     )
 
     execute {
-- 
2.47.2

